src/mkbktr.cpp

```diff
@@ -49,19 +49,19 @@ static void nca_update_ctr(uint8_t *ctr, uint64_t offset) {
 int main(int argc, char **argv) {
   mk::time::Timer _t_main("main()");
 
-  if (argc != 4) {
-    fprintf(stderr, "Usage: %s original patched output\n", argv[0]);
+  if (argc != 5) {
+    fprintf(stderr, "Usage: %s [original nca] [patched nca] [output nca] [/path/to/prod.keys]\n", argv[0]);
     return -1;
   }
 
   // Name args
   const char *original_nca_filename = argv[1];
   const char *new_nca_filename = argv[2];
   const char *bktr_nca_filename = argv[3];
+  const char *path_to_prod_keys = argv[4]; // TODO: this currently needs to be an absolute path.
```

Comment:
Does it? Relative paths should work?

Reply:
just a bad comment, i forgot to remove it